libs/MeshKernel/include/MeshKernel/Parameters.hpp

```diff
@@ -212,6 +212,11 @@ namespace meshkernel
         /// @brief Maximum courant time in seconds
         double max_courant_time = 120.0;
 
+        /// @brief The minimum depth considered when computing refinement based on depths
+        ///
+        /// Only depths greater than this value will be considered.
+        double minimum_refinement_depth = 0.0;
+
```

Comment:
Pass as parameter and remove from the struct here.
This way the api does not change